vizro-core/examples/_visual-vocabulary/pages/spatial.py

```diff
@@ -42,4 +42,81 @@
     ],
 )
 
-pages = [choropleth]
+dot_map = vm.Page(
+    title="Dot map",
+    path="spatial/dot-map",
+    layout=vm.Layout(grid=PAGE_GRID),
+    components=[
+        vm.Card(
+            text="""
+            #### What is a dot map?
+
+            A dot map, or scatter map, uses dots to represent the value of a specific variable at geographic locations.
+
+            &nbsp;
+
+            #### When should I use it?
+
+            Use a dot map to visually display the distribution and concentration of data points across a geographic
+            area. It's ideal for showing the frequency or density of an attribute, helping to identify patterns,
+            clusters, or anomalies.
+
+            Dot maps offer a clear visual impression of spatial distributions, but overlapping dots can make it hard to
+            distinguish individual data points in dense areas. Consider adding opacity to your dots to improve clarity.
+        """
+        ),
+        vm.Graph(
+            figure=px.scatter_map(
```

Comment:
I like the new `scatter_map` 👍 

However, is there any way, and do we actually want to remove this "info" at the bottom right corner of the chart? 

<img width="670" alt="image" src="https://github.com/user-attachments/assets/6e22497e-0d5c-4b68-ae18-26e29103b46e">